test/integration/storage/storage_shared.rb

```diff
@@ -0,0 +1,96 @@
+require "helpers/integration_test_helper"
+require "integration/pubsub/pubsub_shared"
+require "securerandom"
+require "base64"
+require "tempfile"
+
+class StorageShared < FogIntegrationTest
+  def setup
+    @client = Fog::Storage::Google.new
+    # Ensure any resources we create with test prefixes are removed
+    Minitest.after_run do
+      delete_test_resources
+    end
+  end
+
+  def delete_test_resources
+    unless @some_temp_file.nil?
```

Comment:
Don't know why hound is running multiple times, but it runs after every push. Could you add a rule to rubocop yml to remove this output?